libkbfs/backpressure_disk_limiter.go

```diff
@@ -924,30 +997,31 @@ func (bdl *backpressureDiskLimiter) onBlocksFlush(
 	bdl.journalTracker.onBlocksFlush(blockBytes, chargedTo)
 }
 
-func (bdl *backpressureDiskLimiter) onBlocksDelete(
-	ctx context.Context, blockBytes, blockFiles int64) {
+func (bdl *backpressureDiskLimiter) releaseAndCommit(ctx context.Context,
+	typ diskLimitTrackerType, blockBytes, blockFiles int64) {
 	bdl.lock.Lock()
 	defer bdl.lock.Unlock()
-	bdl.journalTracker.onBlocksDelete(blockBytes, blockFiles)
-}
-
-func (bdl *backpressureDiskLimiter) onDiskBlockCacheDelete(
-	ctx context.Context, blockBytes int64) {
-	if blockBytes == 0 {
-		return
+	switch typ {
+	case journalLimitTracker:
+		bdl.journalTracker.releaseAndCommit(blockBytes, blockFiles)
+	case diskCacheLimitTracker:
```

Comment:
Either way is fine with me, I just think the two functions should be consistent.